haskell/private/unix/unix_toolchain.bzl

```diff
@@ -0,0 +1,49 @@
+load("@bazel_skylib//lib:paths.bzl", "paths")
+load(
+    "@rules_haskell//haskell/private/unix:unix_commands.bzl",
+    "unix_commands",
+)
+
+UnixCommandsInfo = provider(
+    doc = "Collection of Unix commands",
+    fields = {
+        cmd: "Absolute path to the {} command or None.".format(cmd)
+        for cmd in unix_commands
+    },
+)
+
+def _unix_toolchain_impl(ctx):
+    commands = {}
+    for cmd in unix_commands:
+        cmd_path = getattr(ctx.attr, cmd, None)
+        if not cmd_path:
```

Comment:
Maybe we should warn if some command is missing as this can lead to hard-to-understand failures at a later stage

Reply:
I think that would be pretty noisy. [That list (IEEE Std 1003.1-2008)](haskell/private/unix/unix_commands.bzl) contains a good number of tools that are not commonly installed on systems. E.g. `fort77`. I expect in most cases missing tools will cause failures in `./configure` scripts which should produce a reasonably clear error message.

Reply:
Weh didn't look closely enough at the list indeed :astonished: